cvat/apps/redis_handler/migration_loader.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (C) 2025 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import importlib
+from datetime import datetime
+from pathlib import Path
+from typing import Any, ClassVar
+from uuid import UUID, uuid4
+
+from attrs import asdict, field, frozen, validators
+from django.apps import AppConfig, apps
+from django.conf import settings
+from redis import Redis
+
+from cvat.apps.redis_handler.redis_migrations import BaseMigration
+
+
+def to_datetime(value: float | str | datetime) -> datetime:
+    if isinstance(value, datetime):
+        return value
+    elif isinstance(value, str):
+        value = float(value)
+
+    return datetime.fromtimestamp(value)
+
+
+def to_uuid(value: str | UUID) -> UUID:
+    if isinstance(value, UUID):
+        return value
+
+    return UUID(value)
+
+
+@frozen
+class AppliedMigration:
+    SORTED_SET_KEY: ClassVar[str] = "cvat:applied_migrations:"
+    KEY_PREFIX: ClassVar[str] = "cvat:applied_migration:"
+
+    name: str = field(validator=[validators.instance_of(str), validators.max_len(128)])
+    app_label: str = field(validator=[validators.instance_of(str), validators.max_len(128)])
+    applied_date: datetime = field(
+        validator=[validators.instance_of(datetime)], converter=to_datetime
+    )
+    identifier: UUID = field(factory=uuid4, converter=to_uuid)
```

Comment:
Why do we need an extra identifier? Shouldn't `<app_label>.<name>` already be a unique key?

Reply:
Yes, it should, probably I did so initially because I would not keep only the applied date in a hash item